src/pipeline.py

```
@@ -49,6 +51,39 @@
 rerank_settings = config.get("ai_reranking_settings", {})
 
 
+class JobAnalysisPipeline:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)